presentation/src/main/java/me/androidbox/presentation/agenda/screen/AgendaScreen.kt

```diff
@@ -43,28 +44,47 @@ fun AgendaScreen(
     agendaScreenEvent: (AgendaScreenEvent) -> Unit,
     onSelectedEditAgendaItemClicked: (id: String, AgendaType, agendaMenuActionType: AgendaMenuActionType) -> Unit,
     onSelectedAgendaItem: (agendaType: Int) -> Unit, /* TODO Check where this is being used */
+    onLogout: () -> Unit,
     modifier: Modifier = Modifier) {
 
     val calendarState = rememberUseCaseState()
 
     Scaffold(
         modifier = modifier,
         topBar = {
-            AgendaTopBar(
-                modifier = Modifier
-                    .fillMaxWidth()
-                    .background(color = MaterialTheme.colorScheme.backgroundBackColor)
-                    .padding(start = 16.dp, end = 16.dp, top = 8.dp, bottom = 8.dp),
-                initials = agendaScreenState.usersInitials,
-                displayMonth = agendaScreenState.selectedDate.month.toString(),
-                onProfileButtonClicked = {
-                    /** TODO Open dropdown menu here */
-                    Log.d("AGENDA_SCREEN", "Profile button clicked")
-                },
-                onDateClicked = {
-                    calendarState.show()
-                },
-            )
+            Column(modifier = Modifier.fillMaxWidth()) {
+                AgendaTopBar(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .background(color = MaterialTheme.colorScheme.backgroundBackColor)
+                        .padding(start = 16.dp, end = 16.dp, top = 8.dp, bottom = 8.dp),
+                    initials = agendaScreenState.usersInitials,
+                    displayMonth = agendaScreenState.selectedDate.month.toString(),
+                    onProfileButtonClicked = {
+                        agendaScreenEvent(AgendaScreenEvent.OnOpenLogoutDropDownMenu(shouldOpen = true))
+                    },
+                    onDateClicked = {
+                        calendarState.show()
+                    },
+                )
+
+                AgendaDropDownMenu(
+                    modifier = Modifier
+                        .background(color = MaterialTheme.colorScheme.dropDownMenuBackgroundColor)
+                        .align(Alignment.End),
```

Comment:
I am trying to get the Dropdownmenu to display at the end below the profile avatar. But for some reason its always displays at the start. I have posted a screenshot in the description. What is the best way to position at the end below the avatar?

Reply:
Have you tried adding a fillMaxWidth to the AgendaDropDownMenu´s modifier?

If that doesn´t work, you can add the AgendaDropDownMenu inside the profile picture composable

Reply:
@MKiperszmid Having the fillMaxWidth doesn't work. Do you have anymore ideas?

![logout](https://user-images.githubusercontent.com/14260802/234455043-7b703de7-6ddd-44e0-8bfe-2040f650abb1.png)

Reply:
I have anchored the dropdownmenu below the whole `AgendaTopBar` using the `Box`. I understand what you mean as the `AgendaTopbar` takes up the whole width of the screen. As in the image below. So the dropdownMenu might display on the left. However, is there another way to position the dropdownmenu on the right without moving the dropdownmenu into the `AgendaTopBar`

![logout-box](https://user-images.githubusercontent.com/14260802/234642986-e1faf92c-a6b3-448f-8c04-dc75580f5952.png)

Reply:
@philipplackner @MKiperszmid 
Uses a composable lambda that the dropdownmenu will be passed to. The result is correct now.

![logoutrightside](https://user-images.githubusercontent.com/14260802/234745906-c9602a27-c75e-44d8-a8e7-77f546409cb3.png)